bip-0003.md

```diff
@@ -0,0 +1,753 @@
+
+Editors do NOT evaluate whether the proposal is likely to be adopted.
+
+A BIP editor will:
+
+* Assign a BIP number and BIP type in the pull request
+* Ensure that the BIP is listed in the [README](README.mediawiki)
+* Merge the pull request when it is ready
+
+The BIP editors are intended to fulfill administrative and editorial responsibilities. The BIP editors monitor BIP
+changes, and update BIP headers as appropriate.
+
+BIP editors may also, at their option, unilaterally make and merge strictly editorial changes to BIPs, such as
+correcting misspellings, fixing broken links, etc. as long as they do not change the meaning or conflict with the
+original intent of the authors. Such a change must be recorded in the Changelog if it’s noteworthy per the criteria
+mentioned in the [Changelog](#changelog) section.
+
+## Backward Compatibility
+
+### Changes from BIP 2
+
+#### Workflow
+
+- Status field values are reduced from nine to four:
+  - Deferred, Obsolete, Rejected, Replaced, and Withdrawn are gathered up into Closed.[^closed]
+  - Final and Active are collapsed into Deployed.
+  - Proposed is renamed to Complete.
+  - The remaining statuses are Draft, Complete, Deployed, and Closed.
+- The comment system is abolished.[^comments]
+- A BIP in Draft or Complete status may no longer be closed solely on grounds of not making progress for three years.[^rejection]
+  - A BIP in Draft status may be set to Closed by anyone if it appears to have stopped making progress for at least a
+    year and its authors do not assert that they are still working on it when contacted.
+  - Complete BIPs can only be moved to Closed by its authors and may remain in Complete indefinitely.
+- Process BIPs are living documents that do not ossify and may be modified indefinitely.
+- Some judgment calls previously required from BIP Editors are reassigned either to the BIP authors or the repository’s
+  audience.
+
+#### BIP Format
+
+- The Standards Track type is superseded by the similar Specification type.[^standard-track]
+- Many sections are declared optional, it is up to the authors and audience to judge whether all relevant topics have
+  been comprehensively addressed and which topics require a designated section to do so.
+- "Other Implementations" sections are discouraged.[^OtherImplementations]
+- Auxiliary files are only permitted in the corresponding BIP’s subdirectory, as no one used the alternative of labeling
+  them with the BIP number.
+- Tracking of adoption, acceptance, and community consensus is out of scope for the BIPs repository, except to determine
+  whether a BIP is in active use for the move into or out of the Deployed status.
+- The distinction between recommended and acceptable licenses was dropped.
+- Most licenses that have not been used in the BIP process have been dropped from the list of acceptable licenses.
+
+#### Preamble
+
+- Comments-URI and Comment-Summary headers are dropped from the preamble.[^comments]
```

Comment:
This comment and the next few harmonize the style in this section; you could choose to go with no quote-wrapping instead.

```suggestion
- The "Comments-URI" and "Comment-Summary" headers are dropped from the preamble.[^comments]
```